torch/csrc/autograd/python_function.cpp

```diff
@@ -387,7 +390,7 @@ static void _wrap_outputs(THPFunction *self, t2var_type &t2var,
   auto cdata = is_volatile ? nullptr : THPFunction_asFunction(self);
   Py_ssize_t num_outputs = PyTuple_GET_SIZE(raw_output);
   if (self->cdata.is_executable) {
-    self->output_info = new std::vector<output_info_type>();
+    self->output_info = new std::vector<VariableInfo>();
```

Comment:
Can you change it to construct the vector inplace? I wrote this code before I new we can use placement new. There's no reason to use the extra pointer